python/lsst/dax/imgserv/getimage/imageget.py

```diff
@@ -371,17 +373,20 @@ def cutout_from_pos(self, params: dict):
         shape = pos_items[0]
         if shape == "BRECT":
             if len(pos_items) < 6:
-                raise Exception("BRECT: invalid number of values")
+                raise UsageError("BRECT: invalid number of values")
             ra, dec = float(pos_items[1]), float(pos_items[2])
             w, h = float(pos_items[3]), float(pos_items[4])
             unit_size = pos_items[5]
             cutout = self.cutout_from_nearest(ra, dec, w, h, unit_size, filt)
             return cutout
         elif shape == "CIRCLE":
             if len(pos_items) < 4:
-                raise Exception("CIRCLE: invalid number of values")
+                raise UsageError("CIRCLE: invalid number of values")
             ra, dec = float(pos_items[1]), float(pos_items[2])
             radius = float(pos_items[3])
+            cutout_area = (radius**2)*math.pi
+            if cutout_area > imgserv_config.max_image_cutout_size:
```

Comment:
I will try to centralize the cutout size checking by looking at the bbox dimensions only, instead of the actual (precise) cutout areas, and that'll take care of BRECT as well.